Package.swift

```diff
@@ -86,11 +87,16 @@ let package = Package(
                 "Bookmarks"
             ]
         ),
-         .target(
-            name: "BloomFilterWrapper",
+        .target(
+            name: "BloomFilterObjC",
             dependencies: [
                 .product(name: "BloomFilter", package: "bloom_cpp")
             ]),
+        .target(
+            name: "BloomFilterWrapper",
```

Comment:
This is the public wrapper in Swift.  It wraps our ObjC wrapper while keeping it private.